req.go

```diff
@@ -581,6 +581,10 @@ func parseCode(id, fileName string, graph reqGraph) error {
 		h.Write([]byte{0})
 	}
 
+	if linepipes.Verbose {
```

Comment:
>  For now, I think a log.Print would do the trick here (or remove the message).

I would rather not print it unconditionally (I think that's what you are suggesting), so I will just remove the print statement.